packages/renderer/src/lib/image/ImageActions.spec.ts

```diff
@@ -19,8 +19,8 @@
 import '@testing-library/jest-dom/vitest';
 import { test, expect, vi, beforeAll } from 'vitest';
 import { render, screen, waitFor, fireEvent } from '@testing-library/svelte';
-import ImageActions from '/@/lib/image/ImageActions.svelte';
-import type { ImageInfoUI } from '/@/lib/image/ImageInfoUI';
+import ImageActions from './ImageActions.svelte';
+import type { ImageInfoUI } from './ImageInfoUI';
```

Comment:
note sure about the why

Reply:
tbh I just instinctively did a 'that's different and longer than it needs to be' and changed it. Re-pushed without this change.